packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.ts

```diff
@@ -143,8 +143,20 @@ export class HorizontalScroll extends FoundationElement {
      * Scrolling state
      * @internal
      */
+    @observable
     private scrolling: boolean = false;
 
+    /**
+     * Firing scrollstart and scrollend events
+     * @internal
+     */
+    private scrollingChanged() {
+        if (this.scrollContainer) {
```

Comment:
does the `this.scrollContainer` check effectively guard against emitting  "scrollend" event on initialization?

Reply:
I found ```scrollingChanged()``` was fired when ```scrollContainer``` was undefined. Presumably during initializing ```scrolling``` being set to false by default.

Reply:
yea, just wanted to make sure it didn't emit a DOM event in that case.